pkg/kubelet/nodeconfig/crashloop.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeconfig
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const startupsFile = ".startups.json"
+const maxStartups = 10
+
+// recordStartup appends a timestamp to the startups-tracking file to indicate a rough Kubelet startup time.
+// If the startups-tracking file cannot be loaded or saved, a fatal error occurs.
+func (cc *NodeConfigController) recordStartup() {
+	const errfmt = "failed to record startup time, error: %v"
+
+	// load the file
+	ls := cc.loadStartups()
+
+	// record current time
+	now := time.Now()
+	stamp := now.Format(time.RFC3339) // use RFC3339 time format
+	ls = append(ls, stamp)
+
+	// rotate the list if necessary
+	if len(ls) > maxStartups {
+		ls = ls[1:]
+	}
+
+	// save the file
+	cc.saveStartups(ls)
+}
+
+// curCrashLooping returns true if the number of startup timestamps since the last modification
```

Comment:
I am ok changing it but I am curious about the reason to avoid these abbreviations. My understanding is that standard Go style is to shorten names where-ever possible, and `cur` seems like a common, well-understood abbreviation. Is this simply to avoid variance in abbreviations (e.g. `th` vs `thr` vs `thresh` etc.), or something else?